src/Factory/ProcessFactory.php

```diff
@@ -23,7 +23,7 @@ class ProcessFactory implements ProcessFactoryInterface
     public function create(string $commandLine): Process
     {
         // See https://github.com/composer/composer/blob/1.10.17/src/Composer/Util/ProcessExecutor.php#L68:L72
-        // @phpstan-ignore-next-line because phpstan can see it's available, but we cannot guarantee symfony >= 4.2 upstream
+        // @phpstan-ignore-next-line because we cannot guarantee symfony >= 4.2 upstream
```

Comment:
I think it is about safe to delete <4.2 logic or are there still Magento 2 projects running on it. Since you're already requiring PHP 8.1 and higher and support for Symfony <4.2 dropped about 4 years ago...

Reply:
I double checked. Turns out the factory is a remnant from 2017 so I removed it entirely. The factory wasn't being used anymore, just instantiated and then unused.